tests/model_formsets/test_uuid.py

```diff
@@ -30,3 +38,135 @@ def test_inlineformset_factory_ignores_default_pks_on_submit(self):
             'uuidpkchild_set-2-name': '',
         })
         self.assertTrue(formset.is_valid())
+
+    def test_inlineformset_factory_nulls_default_pks_uuid_parent_auto_child(self):
+        """
+        #24958 - If we're adding a new object, a parent's auto-generated pk
```

Comment:
Maybe instead of repeating this each time we could say something like "Corresponds to test_inlineformset_factory_nulls_default_pks for the case of ..."